nodejs/week2/homework/meal-sharing/src/backend/data/reviews.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+      "id": 1,
+      "numberOfStars": 3,
+      "content": "Best meal ever",
+      "mealId": 1,
+      "createdAt": 12345
```

Comment:
I will advice you to use proper date, instead of these numbers. You can use the [js date object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) and send the current date.

Reply:
sorry for that. It is should be the right time. :)